test/acceptance/integration/Rules/Cards/card_create_rules.feature

```diff
@@ -201,7 +201,7 @@ Feature: Append a new rule in Perseo manager using cards from portal
     | test1300  | 34       | temper_23      | 12       | SendSmsMibAction | ${device_latitude}${device_longitude}${measure.value} | 123456789       |
     | test1300  | 34       | temper_23      | 0        | updateAttribute  | DANGER                                                | ALARM           |
 
-  @not_updated_card_interval_not_allowed @BUG_53
+  @not_updated_card_interval_not_allowed @BUG_53 @skip
```

Comment:
I see the comment above:

"The skip tag is to skip the scenarios that still are not developed or failed"

However, in this particular case, it should be stated if the test is skipped due to it is not developer or it fails and, in the later case, why is failing.